src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -981,43 +978,33 @@ private static String getFirstDomainsFromIdent(
         /** Import full path. */
         private final String importFullPath;
 
-        /** Import start line number. */
-        private final int startLineNumber;
-
-        /**
-         * Import end line number.
-         * Note: It can be different from <b>startLineNumber</b> when import statement span
```

Comment:
One minor: please move this note to getter Javadoc.

Reply:
@pbludov 
Done.
 *I have not updated the diff report since there is only change in javadoc.* Should I update it?

Reply:
no, this time it is Javadoc only change, is it?